src/core_atmosphere/dynamics/mpas_atm_boundaries.F

```diff
@@ -167,6 +170,7 @@ subroutine mpas_atm_update_bdy_tend(clock, streamManager, block, firstCall, ierr
         ! Compute any derived fields from those that were read from the lbc_in stream
         !
         call mpas_pool_get_array(lbc, 'lbc_u', u, 2)
+        call mpas_pool_get_array(lbc, 'lbc_w', w, 2)
```

Comment:
It's a minor detail, but can we move this below the call to `mpas_pool_get_array` for `lbc_rho_edge` so that the order of variables is consistent in this block and in the block beginning at line 192?

Reply:
Done.